packages/fxa-settings/src/pages/Signin/Signin/en.ftl

```diff
@@ -0,0 +1,17 @@
+# <span> is for visual effect
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - "Continue to" is followed by an image of the logo for the service which the user is authenticating for. alt text for this image is the service name.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo = Continue to { $serviceName }
+signin = Sign in
```

Comment:
```suggestion
signin-button = Sign in
```

I'm assuming this is for the button. Could you adjust the id or add a comment to indicate how this string is being used (otherwise localizers would need to know if its being used as a noun or verb).